releasenotes/v1.10.0.md

```diff
@@ -0,0 +1,112 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.9.0
+
+## :warning: Breaking Changes
+
+- Bump CAPI to v1.10.0-beta.0 (#2432)
```

Comment:
I think we should clean this up so that #2517 is listed here and #2432 is put as superseeded

Reply:
This is the one PR that actually has changes. We could combine them "bump to capi v1.10.0" and list both here, but just adding the simple module bump from rc.1 to stable with no other changes would be very misleading IMO.

Reply:
My issue with this is that it looks like we are bumping to a beta release and then in the fine print we are saying that we actually took he proper release. I know the actual changes happened when we took the first pre-release, but it was still superseded by the later proper release. If we would have skipped the pre-release, then we would definitely list the proper release as breaking, right?